### DISCUSSION TITLE
kiwi_builder_add_word()로 추가 후 지연이 생기는지요?

### DISCUSSION THREAD

**Discussion by CharlesKwonOhjun** [bab2min/Kiwi#123] [Q&A]
안녕하세요?
Windows VIsual C++  19.33.31629 (64bit)에서 CAPI용으로 제공된 DLL을 이용하여 
kiwi_builder_add_word() 함수를 사용하고 있습니다.

kiwi_builder_add_word()로 형태소를 추가하면 0 값을 잘 반환합니다. 
그러나,  kiwi_analyze( ) 호출 후, 결과를 확인하면 랜덤하게 추가된 형태소가 적용되다 안되다 하고 있습니다.
강제로 sleep으로 강제 지연을 주면 대부분 잘 되고 있습니다. kiwi_builder_add_word() 후에 추가 사용 함수가 따로 있는지요?

감사합니다.

**Comment by CharlesKwonOhjun** [comment]
함수 호출 순서를 바꾸어 해결된 것 처럼 보였지만 
default.dict 에 물리적으로 추가해서 결과가 나온 것으로 확인 되었습니다.
프로그램 내에서 kiwi_builder_add_word() 실행시 램덤하게 반영되는 부분은 어디를 살펴 봐야 할지 문의 드립니다.

**Reply by bab2min** [reply]
안녕하세요 @CharlesKwonOhjun 사용하신 코드를 공유해주시면 문제를 확인하는데에 도움이 될것 같습니다.
참고로 `kiwi_builder_add_word`를 사용하는 예시는 아래 코드와 같습니다.
https://github.com/bab2min/Kiwi/blob/6713d4ffc069046aa000845daa4e4017d7ba3ab0/test/test_c.cpp#L29-L45
